Makefile

```diff
@@ -111,17 +114,16 @@ clean:
 
 local-image:
 ifdef USE_BUILDAH
-	buildah bud $(BUILDAH_OPTS) -t $(IMAGE_TAG) -f $(DOCKERFILE) .
+	buildah bud $(BUILDAH_OPTS) -t $(IMAGE) -f $(DOCKERFILE) .
 else
-	sudo docker build -t $(IMAGE_TAG) -f $(DOCKERFILE) .
+	$(SUDO_CMD) $(CONTAINER_ENGINE) build -t $(IMAGE) -f $(DOCKERFILE) .
```

Comment:
If we need `$(SUDO_CMD)` here, do we also need it for `buildah`?  Or should we just drop it for podman?  Or is the intent to still support `docker` via this?